generic_config_updater/generic_updater.py

```diff
@@ -19,7 +19,9 @@ def extract_scope(path):
     if not path:
         raise Exception("Wrong patch with empty path.")
     pointer = jsonpointer.JsonPointer(path)
-    parts = pointer.parts
+
+    # Re-escapes `/` in path parts by converting `/` back to `~1`.
+    parts = [part.replace("/", "~1") for part in pointer.parts]
```

Comment:
Is it a issue for multi-asic only? 
I remember in single asic, the jsonpointer has built in feature to parse / to ~1

Reply:
It was introduced by multi asic change, we have to extract the namespace from the patch, then JsonPointer will do escape `~1` back to `/`

Reply:
Sorry. I don't get it. 
By the code here, the extract_scope doesn't differentiate single or multi. How do we guarantee single asic is not broken? https://github.com/sonic-net/sonic-utilities/blob/319f58d5d5dbe86e2f0aeece67b8274dcd3f3ce8/config/main.py#L1599-L1600

Reply:
For single asic, the portchannel interface will not having slash in port, but multi asic does have, hence, we could do escape slash to ~1 in jsonpatch.

Reply:
Another reason is single asic logic is taking path as input parameter directly, but multi asic, we have to do combine with parts by Jsonpointer result..